fr.md

```diff
@@ -1,58 +1,62 @@
-Nous utilisons un service de tiers appelé « App Center » pour collecter et analyser des rapports d’erreur de notre application Android. Ceux-ci contiennent des données telles que le numéro de référence de l’application, le modèle de l’appareil et le système d’exploitation, qui nous permettent de corriger les bugs.
-Pour en savoir plus sur la manière dont le « App Center » traite vos données, consultez la [Politique de confidentialité de Microsoft](https://privacy.microsoft.com/fr-fr/privacystatement).
+Les attaques de spam sont coûteuses et peuvent détourner des fonds importants de nos efforts de plantation d'arbres. Par conséquent, nous gérons notre propre intelligence de protection contre le spam, mais nous demandons également l'aide du spécialiste de la protection contre le spam, Cloudflare. En interne, nous conservons les adresses IP pendant un maximum de 7 jours avant de les anonymiser ou de les supprimer. Cloudflare reçoit des adresses IP anonymisées et obfusquées, dont la plupart sont supprimées en quelques jours. Seul un petit échantillon de métadonnées anonymisées reste chez Cloudflare pendant une période pouvant aller jusqu'à 12 mois. Cet échantillon est conservé afin de diagnostiquer les activités malveillantes et d'améliorer l'efficacité des mesures de sécurité. Pour plus d'informations, veuillez consulter [la politique de confidentialité de Cloudflare](https://www.cloudflare.com/fr-fr/privacypolicy/).
 
-## Campagnes marketing
+### Applications mobiles
 
-Afin de planter plus d’arbres, nous avons recours à des plateformes publicitaires externes pour toucher de nouveaux utilisateurs. Lorsque les nouveaux utilisateurs installent l’extension Ecosia sur leur ordinateur pour la première fois, une page s’affiche et informe nos partenaires publicitaires qu’Ecosia vient d’être installée. Cette page n’est pas reliée à notre service de recherche pour protéger votre vie privée. Elle nous permet d’identifier les publicités qui nous ont aidé à planter des arbres et d’éviter de dépenser plus d’argent pour des campagnes inutiles. Nous testons de nouvelles plateformes de manière régulière et supprimons celles qui s’avèrent inefficaces. Nous utilisons ou nous prévoyons actuellement de tester [Google Ads](https://policies.google.com/privacy?hl=fr&gl=fr), [l’App Store d’Apple](https://developer.apple.com/app-store/app-privacy-details/), [Facebook](https://fr-fr.facebook.com/privacy/explanation/), [Microsoft Bing](https://about.ads.microsoft.com/fr-fr/policies/legal-privacy-and-security), [Twitch](https://www.twitch.tv/p/fr-fr/legal/privacy-notice/) et [TikTok](https://www.tiktok.com/legal/privacy-policy-eea).
+Pour améliorer nos applications de navigation, nous recueillons des statistiques d'utilisation telles que le nom de l'appareil, son fabricant, son modèle, le numéro du système d'exploitation et le numéro de l'application ou la version du SDK.
 
```

Comment:
I think we do not need an empty line here

Reply:
Thanks, I had thought this was a formatting error!